Source/Pawnmorphs/Esoteria/MorphUtilities.cs

```diff
@@ -535,29 +535,30 @@ public static bool ShouldBeConsideredHuman([NotNull] this Pawn pawn)
 
         private static MorphDef GetChimeraRace([CanBeNull] AnimalClassBase hInfluence)
         {
-            if (hInfluence == null)
+            if (hInfluence != null)
             {
-                return MorphCategoryDefOf.Chimera.AllMorphsInCategories.RandomElement();
-            }
-
-            var morph = hInfluence as MorphDef;
-            //if the highest influence isn't a morph pick a random morph from the animal class
-            morph = morph ?? ((AnimalClassDef) hInfluence).GetAllMorphsInClass().RandomElementWithFallback();
-            
-            // This can happen if the AnimalClassDef doesn't have any associated morphs
-            if (morph == null)
-                return MorphCategoryDefOf.Chimera.AllMorphsInCategories.RandomElement();
+                var morph = hInfluence as MorphDef;
+                //if the highest influence isn't a morph pick a random morph from the animal class
+                morph = morph ?? ((AnimalClassDef) hInfluence).GetAllMorphsInClass().RandomElementWithFallback();
             
-            if (morph.classification.Contains(AnimalClassDefOf.Canid)) //TODO Generalize this or just pick randomly, 
-                return MorphDefOfs.ChaofoxMorph;
-            if (morph.classification.Contains(AnimalClassDefOf.Reptile))
-                return MorphDefOfs.ChaodinoMorph;
-            if (morph == MorphDefOfs.BoomalopeMorph) return MorphDefOfs.ChaoboomMorph;
-            if (morph == MorphDefOfs.CowMorph) return MorphDefOfs.ChaocowMorph;
+                // This can happen if the AnimalClassDef doesn't have any associated morphs
```

Comment:
Since you inverted this if, the comment should probably be clarified that it's referring to morph being null.  Not a huge deal, though.

i.e. `// Morph can be null here if the AnimalClassDef doesn't have any associated morphs`